app/templates/admin/services/service_list.html

```diff
@@ -0,0 +1,33 @@
+{% extends 'admin/base.html' %}
```

Comment:
General note about structuring the template files here, I like that you created a specific directory for `admin` but I think no need to have a separate directory per module, like services templates here can be on the same level inside `admin` and no need to create `services` directory